modules/virt-adding-static-public-key-web.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+[id="adding-static-public-key-web_{context}"]
+= Adding a static public key by using the web console
+
+Add a static public SSH key to a virtual machine (VM) by using the {product-title} web console. The key is saved as a secret in the project.
+
+[IMPORTANT]
+====
+You must add the key to the VM _before_ it boots for the first time.
+
+Only the user who created the SSH key pair can access the VM.
```

Comment:
I think this note is not necessary and we could drop it.

Reply:
You mean dropping the sentence about the user who created the SSH key pair? If another user tries to use "virtctl SSH" to the VM and does not have the private SSH key, the command will not work.